lib/perl/Genome/Model/Tools/Analysis/Coverage/CoveragePlot.R

```diff
@@ -1123,6 +1123,9 @@ max1 <- 0;
 for (i in 1:length(b$stats))
     max1 <- max(max1, as.numeric(unlist(strsplit(b$stats[i], ","))));
 
+# if with extreme distribution
+if (max1 < 5) max1 <- 5;

```

Comment:
Would it make sense to replace `max1 <- 0` before the `for` loop with `max1 <- 5`, and then not add this conditional?